cfgov/v1/models/sublanding_filterable_page.py

```diff
@@ -46,8 +49,22 @@ class ActivityLogPage(SublandingFilterablePage):
 
     objects = PageManager()
 
-    def base_query(self, hostname):
-        return AbstractFilterPage.objects.live_shared(hostname)
+    @classmethod
+    def eligible_categories(cls):
+        categories = dict(ref.categories)
+        return sorted(itertools.chain(*(
+            dict(categories[category]).keys()
+            for category in ('Blog', 'Newsroom', 'Research Report')
+        )))
```

Comment:
The subcategory mappings are listed [here](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/util/ref.py#L51).